docs/upgrading/component-routes.md

```diff
@@ -245,7 +245,37 @@ export default function Component() {
 
 [View our guide on configuring routes][configuring-routes] to learn more about the `routes.ts` file.
 
-## 7. Boot the app
+### 7. Configure Typegen
```

Comment:
```suggestion
### 7. Enable type safety
```
Matching the remix upgrade doc: https://reactrouter.com/upgrading/remix#7-enable-type-safety